### DISCUSSION TITLE
[Feature] add indicator about existing geo data on thumbnails and big size view

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6190] [Feature Request]
### The feature

I want to quickly see, if an asset has geo coordinates or not. There should be an indicator on thumbnails and in the one-asset-view.

Why?
- timezone correction uses existing geo data. This indicator could help to understand wrong order
- missing geo data could be easy identified in the timline, to be addited

Bonus:

The indicator could be click able to go directly to the coordinates / to the geo editor / location tags / map or something similar.

Also on read only external libraries it should be possible to get the quick geo info.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by aisbergde** [comment]
related feature:
copy and paste geo data from one thumbnail into another

**Comment by bo0tzz** [comment]
Adding indicators like this to the main gallery view can easily get out of hand. IMO a better solution will be to add an option to the search to filter for assets that (don't) have this data.

**Reply by aisbergde** [reply]
Maybe you are right. The main issue is #6189 an indicator could help. I don't think, that this could be a performance issue. And the copy and paste could just help to solve issues with missing geo data. That's why it is important to see content with and without geo data in the timeline.

Currently, I use the https://docs.digikam.org/en/geolocation_editor.html in digiKam.

**Comment by aisbergde** [comment]
I was in the wrong feature with my comment, sorry

**Comment by bo0tzz** [comment]
#21029
